src/pages/Home.jsx

```diff
@@ -0,0 +1,85 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import ArticleCard from "../components/ArticleCard";
+import Error from "./Error";
+import { Loader } from "lucide-react";
+import InfiniteScroll from "react-infinite-scroll-component";
+import Navbar from "../components/Navbar";
+import { Link } from "react-router-dom";
+
+const Home = () => {
+  const [page, setPage] = useState(2);
+  const [items, setItems] = useState([]);
+  const [error, setError] = useState(null);
+  const [hasMore, setHasMore] = useState(true);
+
+  useEffect(() => {
+    const getArticles = async () => {
+      const res = await axios.get(
+        `https://be-news-api-h65m.onrender.com/api/articles?p=1&limit=5`,
+      );
+      setItems(res.data.articles);
+    };
+    getArticles();
+  }, []);
+
+  const fetchArticles = async () => {
+    const res = await axios.get(
+      `https://be-news-api-h65m.onrender.com/api/articles?p=${page}&limit=5`,
+    );
+    return res.data.articles;
+  };
+
+  const fetchData = async () => {
+    const articlesFromServer = await fetchArticles();
+
+    setItems([...items, ...articlesFromServer]);
+    if (articlesFromServer.length === 0 || articlesFromServer.length < 5) {
+      setHasMore(false);
+    }
+    setPage(page + 1);
+  };
+
+  return (
+    <>
+      <Navbar></Navbar>
```

Comment:
should each article have a navbar? if the navbar should be seen on every single page then this should go into app.jsx